tests/userdashboard/test_serializers.py

```diff
@@ -9,35 +9,46 @@
 
 
 @pytest.mark.django_db
-def test_num_reports(apiclient, report_factory, comment_factory, idea):
-    comment_1 = comment_factory(content_object=idea)
-    report_factory(content_object=comment_1)
+def test_num_reports(
+    apiclient,
+    report_factory,
+    ai_report_factory,
+    comment_factory,
+    idea,
+):
+    comments = comment_factory.create_batch(size=4, content_object=idea)
+    comments = {comment.pk: comment for comment in comments}
+    pks = list(comments)
 
-    comment_2 = comment_factory(content_object=idea)
-    report_factory(content_object=comment_2)
-    report_factory(content_object=comment_2)
+    n_user_reports = [1, 3, 0, 0]
+    num_user_reports_created = {pk: size for pk, size in zip(pks, n_user_reports)}
```

Comment:
more of a style choice I guess but this could also simply be `dict(zip(pks, n_user_reports))`

Reply:
good point, done.